chill-scala/src/main/scala/com/twitter/chill/ScalaKryoInstantiator.scala

```diff
@@ -201,5 +202,13 @@ class AllScalaRegistrar extends IKryoRegistrar {
     val boxedUnit = scala.Unit.box(())
     k.register(boxedUnit.getClass, new SingletonSerializer(boxedUnit))
     PackageRegistrar.all()(k)
+
+    // Enable Java 8 lambda serialization only if we are running on a Java 8 JRE:
```

Comment:
Whoops, forgot about this comment until now. I've created a `Java8ClosureRegistrar` to abstract away this code.